### ISSUE TITLE
Lay down the Road map (Update README.md).

### ISSUE THREAD

**Issue by rajcrk** [fnplus/initDev#7]
Hey, @xlogix I'm really interested in the initial idea that you had for this project.

> I had this idea for creating a front-end for this project which could allow users to drag and drop what they need like Scratch or Google's Blockly. 

It would be a lot more awesome if we could set ourself the <b>Long term ROADMAP</b> for this project. (Having a backend, frontend that kinda stuff!) Inviting more people to contribute to this project as well.

This can be an open discussion about how we can proceed further, What frameworks, tools, blah, blah... that we could use.

I guess this can be just "baby steps" towards something hopefully amazing 😎  and we could hopefully start out by laying out the plan in the README.md document.

**Comment by xlogix** [comment]
I'm not really good at this stuff but I'll try. Can you refer me a repo to look at?

**Comment by rajcrk** [comment]
@xlogix I`m not good at it either. But I think this would be something that we could look at to get started.
https://github.com/vuejs/roadmap/blob/master/README.md

We could just look into the core of the above document, I think that might be just enough to get us started. Do let me know what you think.

**Comment by xlogix** [comment]
Alright thanks!

**Comment by rajcrk** [comment]
Hey @xlogix , I have a few ideas in my mind to get started. I'm quite familiar with using [NestJs](https://nestjs.com/) as a back-end framework for this application.

Maybe we could start out by building a bare backend for this application and see how we can go from there. What's your thought?

**Comment by xlogix** [comment]
Yes, sure. I'll be putting things down in the roadmap as well (as WFH is given now)
